pp_sketch/__main__.py

```diff
@@ -145,6 +145,14 @@ def get_options():
         else:
             arguments['--kNN'] = 0
             arguments['--threshold'] = float(arguments['--threshold'])
+        if arguments['--require-reciprocity']:
+            arguments['--require-reciprocity'] = True
+        else:
+            arguments['--require-reciprocity'] = False
+        if arguments['--count-all-neighbours']:
+            arguments['--count-all-neighbours'] = True
+        else:
+            arguments['--count-all-neighbours'] = False
```

Comment:
You don't need this, you can just use `if arguments['--require-reciprocity']` later

Reply:
Gone.